electron_app/src/ipc/login.js

```diff
@@ -0,0 +1,75 @@
+const ipc = require('@criptext/electron-better-ipc');
+const {
+  extractResponseSubset,
+  rejectAsNetworkError
+} = require('../utils/httpUtils');
+const { hashPassword } = require('../utils/HashUtils');
+const { getComputerName } = require('../utils/osUtils');
+const {
+  DEV_SERVER_URL,
+  PROD_SERVER_URL,
+  DEVICE_TYPE
+} = require('../utils/const');
+const ClientAPI = require('@criptext/api');
+
+const client = new ClientAPI({
```

Comment:
use client from clientManajer.js

Reply:
I decided to not use `clientManager.js` because that class always checks for a token. This is login, there will never be a token so there's no need to do that.

Reply:
I decided use only one file to access client :) Please follow this